recipe/session/accessTokenVersions_test.go

```diff
@@ -184,14 +184,21 @@ func TestShouldThrowErrorWhenUsingProtectedProps(t *testing.T) {
 	}
 	res2, err2 := http.Post(testServer.URL+"/create", "application/json", bytes.NewBuffer(postBody))
 	if err2 != nil {
```

Comment:
test should be to create a session (add a custom prop), and then create another session by copying over all the access token payload of the previous session (new one should have the same custom prop)